x-pack/plugins/ml/public/application/util/time_series_explorer_service.ts

```diff
@@ -0,0 +1,646 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
Let's add a comment here like:

```
// TODO Consolidate with legacy code in
// `ml/public/application/timeseriesexplorer/timeseriesexplorer_utils/timeseriesexplorer_utils.js`.
```

Reply:
All comments for consolidation added in 1bd6b8bfce5ac81e7c14e47c074387493e0bc075